about.html

```diff
@@ -0,0 +1,197 @@
+					   inputs and solutions international trade fair in Ethiopia.
+					</p>
+					<p>
+						This event has proved to be the major platform that fuels the sector development through trade facilitation, market
+						linkages, introduction of new technology and transfer of knowledge for concerned stakeholders across the whole value
+						chain.
+					</p>
+				</div>
+				<div class="headline-contact">
+					<p>
+						Please contact us per Email for any further questions about Ethio Health Exhibition 2023!
+					<br>
+						<a>marketing@pranevents.net</a>
+					</p>
+				</div>
+			</div>
+		</div>
+	</header>
+	<main class="container">
+		<h2 class="main-title">
+			Ethio Health Exhibition and Congress Logo 2023 Logo
+		</h2>
+		<p class="main-paragraph">
+			Ethio Health Exhibition and Congress Logo 2023 was decided through the logo
+			competition from 8. June to 7. July.
+		</p>
+		<p class="main-paragraph-desktop">
+			Ethio Health Exhibition and Congress Logo 2023 was decided through the logo
+			competition from 8. June to 7. July. After online public votting and selection commitee's judging process,
+			the logo from Naresh Agrawal won the prize.
+			After consultation with Mr. Agrawal we optimised the prize-winning-logo for this years's summit as following:
+		</p>
+		<div class="main-logo">
+			<img src="./img/logo/ethio-health-logo.png" alt="Ethio health logo">
+		</div>
+	</main>
+	<section class="container past-events">
+		<h2 class="past-events-title">
+			See the past Ethio Health Exibition
+		</h2>
+		<div class="past-events-description">
+			<p>
+				Take a look at the last two Ethio Health Exhibition which took place
+				in Addis Ababa, Ethiopia.
+			</p>
+		</div>
+		<div class="past-event-cards">
+			<div class="past-event-card-1">
+				<h2 class="past-event-date">
+					2021
+				</h2>
+				<p class="past-event-site">
+					Ethio Health Exhibition 2021 in Addis Ababa, Ethiopia
+					
+				</p>
+			</div>
+			<div class="past-event-card-2">
+				<h2 class="past-event-date">
+					2022
+				</h2>
+				<p class="past-event-site">
+					Ethio Health Exhibition 2022 in Addis Ababa, Ethiopia	
+				</p>
+			</div>
+		</div>
+	</section>
+	<footer>
+		<div class="partners">
+			<div class="partner-title">
+				<p>Partner</p>
+			</div>
+			<div class="container partner-group">
+				<img src="img/logo/prana-events.png" alt="" class="part">
+				<img src="img/logo/moh.png" alt="" class="part">
+				<img src="img/logo/ELA-logo.png" alt="" class="part">
+				<img src="img/logo/mice-logo.png" alt="" class="part">
+				<img src="img/logo/Efda-logo.png" alt="" class="part">
+			</div>
+		</div>
+		<div class="footer-right-reserved">
+			<div class="container-footer-reserved">
+				<div class="footer-logo">
+						<img src="./img/logo/ethio-health-logo.png">
+				</div>
+				<p class="footer-paragraph-right-reserved">
+						Ⓒ 2023 Prana Events, All rights reserved.
+				</p>
+			</div>
```

Comment:
- [ ] Please fix the footer here and make sure to add a space between the logo and the text.
![image](https://user-images.githubusercontent.com/74744787/218277801-0e15e95a-0865-455c-9b9a-ca94b54497d8.png)